### PR TITLE
Initial commit to html-css-popup

### PR DESCRIPTION
**Pull request by Jigsaff**
### html-css-popup

[Demo](https://jigsaff.github.io/html-css-popup/) | [Code base](https://github.com/Jigsaff/html-css-popup/)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by lenadgit** [comment]
Great job!  But if the user will use a Tab key for MENU navigation - open a popup will be impossible. Moving by Tab key - is a requirement for [a11y](https://developer.mozilla.org/en-US/docs/Web/Accessibility). Could you please fix it?

File: submissions/Jigsaff/html-css-popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+body {
+    background-color: #14394C;
+    background-image: url(../img/bg.jpg);
+    /*background-position-y: bottom;
+    background-position-x: center;*/
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+a:visited,
```
Comment (lenadgit) [inline submissions/Jigsaff/html-css-popup/css/style.css:24]: and the same for link (as on line 10)

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Comment by Jigsaff** [comment]
Fixed, please review!

**Review by lenadgit — CHANGES_REQUESTED** [review]
When the user chooses the popup icon - the popup window should appear by clicking the Space key. Now there is not a possibility to open this popup by keyboard. Could you please fix it?

**Comment by Jigsaff** [comment]
Fixed, please review!

**Review by lenadgit — APPROVED** [review]
Thank you for the submission. Feel free taking the next course module while you submission is being reviewed

**Pull request merged**